packages/reporting-e2e/docker/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.5'
+services:
+  keycloak:
+    build: keycloak
+    image: rmf-web-e2e/keycloak
+    ports:
+      - 8088:8080
```

Comment:
If we want to run locally the `dashboard-e2e` and the `reporting-e2e` at the same time, I think we can have problems because the keycloak has the same port.

Reply:
oh hmmm. this might be good to add into the new [issue](https://github.com/open-rmf/rmf-web/issues/439)
maybe we can discuss how to separate the common components / utils as well

Reply:
Yeah, makes sense!